README.md

```diff
@@ -107,10 +107,17 @@ are supported using the folder structure above.
   <summary> <b> HDD </b> </summary>
 <p>
 
-For PS2, 48-bit LBA internal HDDs up to 2TB are supported. HDD should be
-formatted with the APA partition scheme. OPL will create the `+OPL` partition on the HDD.
-To avoid this, you can create a text file at the location `hdd0:__common:pfs:OPL/conf_hdd.txt`
-that contains the preferred partition name (for example `__common`).
+For PS2, 48-bit LBA internal HDDs up to 2TB are supported. The HDD should be
+formatted with the APA partition scheme, recommended tools are listed below. OPL will create the `+OPL` partition.
+To avoid this, before running OPL the first time, create a text file at the location `hdd0:/__common/OPL/conf_hdd.txt`
```

Comment:
Not your fault, it’s wrong in the original but the file is actually `conf_hdd.cfg` not .txt